tasks.py

```diff
@@ -0,0 +1,47 @@
+import sys
+from invoke import task
+
+
+@task
+def benchmark(c):
+    """CI benchmark"""
+    c.run(
+        "pytest --benchmark-only --benchmark-group-by=param:h5path --benchmark-columns=median,iqr,min,max"
+    )
+
+
+@task
+def black(c):
+    c.run(f"{sys.executable} -m black --check h5grove/ example/ test/")
+
+
+@task
+def flake8(c):
+    c.run(f"{sys.executable} -m flake8 -v")
+
+
+@task
+def mypy(c):
+    c.run(f"{sys.executable} -m mypy h5grove/ example/ test/")
+
+
+@task
+def lint(c):
+    """Lint"""
+    black(c)
+    flake8(c)
+    mypy(c)
+
+
+@task
+def test(c, benchmark=False):
+    """Test with or without benchmark"""
+    c.run(f"pytest --cov=h5grove {'' if benchmark else '--benchmark-skip'}")
```

Comment:
Since there is a `benchmark` command, I'd drop the benchmark option here:
```suggestion
def test(c):
    """Test without benchmark"""
    c.run("pytest --cov=h5grove --benchmark-skip")
```

and maybe also the `--cov=h5grove` since it it already in `pyproject.toml` to avoid duplicated config:

```suggestion
def test(c):
    """Test without benchmark"""
    c.run("pytest --benchmark-skip")
```

https://github.com/silx-kit/h5grove/blob/8867c17cfeb72301d5944e1ad5698a03efe9212e/pyproject.toml#L7